src/Orchard.Web/Modules/Orchard.Projections/Views/EditorTemplates/Parts/QueryPart_Edit.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@model Orchard.Projections.ViewModels.QueryViewModel
+@using Orchard.Projections;
+
+<fieldset>
+    @Html.LabelFor(m => m.VersionScope, T("Query Version Scope"))
```

Comment:
"Version Scope" is not very intuitive. "Versions" only seems better. What do we use in Lucene indices?

Reply:
Ok for "Versions", or "Content versions"?